hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -162,9 +162,10 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
     /**
      * @param readCount Number of items in the response.
```

Comment:
Updated in the other PR : https://github.com/hazelcast/hazelcast-client-protocol/pull/87